src/components/MoviesList/MoviesList.tsx

```diff
@@ -2,14 +2,29 @@ import React from 'react';
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
+interface Movie {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string
+}
+
 interface Props {
   movies: Movie[];
+  value: string;
+}
+
+function visibleMovies(item:Movie, input:string) {
+  return item.title.toLowerCase().includes(input.toLocaleLowerCase())
+    || item.description.toLowerCase().includes(input.toLocaleLowerCase());
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
+export const MoviesList: React.FC<Props> = ({ movies, value }) => (
   <div className="movies">
     {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
+      visibleMovies(movie, value)
+        && <MovieCard key={movie.imdbId} movie={movie} />
```

Comment:
Implementation is working, but I consider you can do it easier. Have a look at my code.

Reply:
+1
you can filter your `moviesFromServer` in the parent component, and here you won't need any additional logic 😉